### PR TITLE
Add API tests

### PR DESCRIPTION
**Pull request by Abdulkadir98**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,9 +2,6 @@ const express = require('express');
 const bodyParser = require('body-parser');// initialize our express app
 const app = express();
 
-//set port number
-let port = 1234;
```
Comment (patelvp) [inline app.js]: Does this change port from 1234 to 3000?

Comment (Abdulkadir98) [reply app.js]: yes actually the entry point of the app is in /bin/www The server starts in that file with port 3000. Somebody has added this line and it keeps getting merged

Comment (patelvp) [reply app.js]: Hopefully does not brake deployment

**Comment by patelvp** [comment]
Waiting for build to pass

**Comment by patelvp** [comment]
Tests failing  again

**Comment by patelvp** [comment]
@Abdulkadir98 Wanted to debug this on production hence the console.logs

**Comment by Abdulkadir98** [comment]
@nehagundecha95 I pulled from develop and merged the changes

**Comment by patelvp** [comment]
overriding sonar check since pr is urgently required.

**Pull request merged**